Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybookToUpdateSeverity.json

```diff
@@ -0,0 +1,406 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "",
+        "description":  "",
+        "prerequisites":  "",
+        "prerequisitesDeployTemplateFile":  "",
+        "lastUpdateTime":  "",
+        "entities":  [
+        ],
+        "tags":  [
+        ],
+        "support":  {
+            "tier":  "community",
+            "armtemplate":  "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
+        },
+        "author":  {
+            "name":  ""
+        }
+    },
+    "parameters":  {
+        "PlaybookName":  {
+            "defaultValue":  "Digital Shadows Playbook to Update Incident Severity",
```

Comment:
I thought we had changed the playbook so that it only updated the Status of the incident and that severity mapping was done in the Analytic Rule.

Reply:
when we fix this can we also rename the file please.